### PR TITLE
home work w1 d3

### PR DESCRIPTION
**Pull request by sultan2001b**
How difficult did you find this (out of 10)? 2
Was there anything that you struggled with?; no
Is there anything that you'd like some further information on?; no
Roughly how long did it take? 45m

### CODE REVIEW

**Comment by sultan2001b** [comment]
comment for home work w1 d4
How difficult did you find this (out of 10)? 
4
Was there anything that you struggled with?; 
i had some bugs, but fixed directly
Is there anything that you'd like some further information on?;
no thanks
Roughly how long did it take?
1:30

File: Sultan_Alsamaani/week_01/day_03/main.js [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+}
+function areaOfCircle(num) {
+    return 3.14159*squareNumber(num);
+}
+
+function part2(num) {
+    var result = halfNumber(num);
+    result = squareNumber(result);
+    result = areaOfCircle(result);
+    result = percentOf(result,squareNumber(result));
```
Comment (mfalthaw) [inline Sultan_Alsamaani/week_01/day_03/main.js]: This is great! The only problem I see is that you're overriding the value of `result` every time you call the next function.

File: Sultan_Alsamaani/week_01/day_03/main.js [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+    return result;
+}
+//******************/
+
+function cl(text) {
+    console.log(text);
```
Comment (mfalthaw) [inline Sultan_Alsamaani/week_01/day_03/main.js]: I like this!

File: Sultan_Alsamaani/week_01/day_04/main.js [hunk 3]
```diff
@@ -0,0 +1,139 @@
...
+3. Write a function findLongestWord that takes an array of words and returns the length of the longest word in the array.
+*/
+function findLongestWord(arr) {
+    var index = 0;
+    for (var i = 0; i < arr.length; i++) {
+        if (arr[i].length >= arr[index].length) {
+            index = i;
+        }
+    }
+    return arr[index].length;
```
Comment (mfalthaw) [inline Sultan_Alsamaani/week_01/day_04/main.js]: Great!

File: Sultan_Alsamaani/week_01/day_03/main.js (hunk 1 above)
Comment (sultan2001b) [reply Sultan_Alsamaani/week_01/day_03/main.js]: 'overriding the value of result'
it similar like this code
var halfNumberResult = halfNumber(num);
 var   squareNumberResult = squareNumber(halfNumberResult);
 var   areaOfCircleResult = areaOfCircle(squareNumberResult);
  var  FinalResult = percentOf(result, squareNumber(areaOfCircleResult));

or in another way
var result =percentOf(areaOfCircle(squareNumber(halfNumber(num))))

advice me please how should i do it
thanks

**Pull request merged**